### ISSUE TITLE
No icons are created

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nico/docerator#2]
```
What steps will reproduce the problem?
1. Selected an .icns file containing my app icon
2. Pressed Create Icons and selected a folder

What is the expected output? What do you see instead?

I expected to have the icons created, but no icons were written to disk

What version of the product are you using? On what operating system?

1.0 on OS X 10.8.3
```

Original issue reported on code.google.com by `ericg...@gmail.com` on 23 May 2013 at 9:03

**Comment by sveinbjornt** [comment]
I've fixed the issues discussed here in the docerator codebase. I'm going to try to find time to clean it up and share but in the meantime you can msg me if you want a modern version that works on retina Macs and the latest versions of OS X, and generates 512x512@2x etc.

**Comment by DanielGibbsNZ** [comment]
@sveinbjornt I'd definitely be interested in a modern version. Any plans to make it publicly available?

**Comment by sveinbjornt** [comment]
Yeah, that was the plan. It's in a messy state at the moment but I can send it to you anyway if you just need something that works.

**Comment by DanielGibbsNZ** [comment]
That would be much appreciated, thanks!
